kernel/src/process_utilities.rs

```diff
@@ -247,3 +269,30 @@ pub fn load_processes<C: Chip>(
 
     Ok(())
 }
+
+/// This is a wrapper function for `load_processes_advanced` that uses
+/// the default arguments that mainstream boards should provide.
+///
+/// Default arguments are:
+///  - `enforce_kernel_version`: prevent loading processes that do not provide a `KernelVersion`
```

Comment:
```suggestion
///  - `require_kernel_version`: prevent loading processes that do not provide a `KernelVersion`
```

Reply:
This was marked as resolved but does not look to be changed?